redis/src/cluster.rs

```diff
@@ -686,9 +686,12 @@ impl NodeCmd {
     }
 }
 
+/// TlsMode indicate tls mode of every connections of ClusterConnection.
 #[derive(Clone, Copy)]
-enum TlsMode {
+pub enum TlsMode {
+    /// Secure means every connections of ClusterConnection are connected over tls.
```

Comment:
I don't think these comments represent what these were originally meant to mean -- at least I was under the impression this was about certificate verification being enabled when using TLS. Which is it? Is this different from non-cluster parts of the library?

Reply:
You can see the correct meaning of both in the original PR: https://github.com/redis-rs/redis-rs/pull/670/files#diff-3081b035ea7b3a6c6fbf36bdcc857b7d1405e36fbe98df550dc7ba187275a638R66.

Reply:
Yes, you are correct. It represents do not check certificate verification. I'll change comment.
https://github.com/redis-rs/redis-rs/blob/8eda667d087967d049d238f538c640b13ac69c6e/redis/src/connection.rs#L54-L62